lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -464,11 +484,12 @@ mapping:
 
       mulled_channels:
         type: str
-        default: 'conda-forge,bioconda'
```

Comment:
I think this is actually the default value:

https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L453

If you restore this, you can simplify `lib/galaxy/config/__init__.py` I think.

Reply:
It's one of the edits I wasn't sure about. If we do this (I agree, it makes perfect sense), we should also remove this: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/deps/mulled/mulled_build.py#L42
and 8 occurrences in 3 other files. If you agree, I'll do this as a separate PR.

Reply:
and instead of those occurrences, simply use the value loaded at startup (i.e., the schema default unless overwritten) that is stored in app.config

Reply:
Oh, I see. So those channels would be specified in 2 places.. On the other hand that gets rid of an import; and that value, I suppose, would very rarely need to be modified, if ever. Yes, I guess that's a reasonable tradeoff. Changing.